specs/deposits.md

```diff
@@ -296,6 +296,16 @@ Total calldata length MUST be exactly 164 bytes, implying the sixth and final se
 partially filled. This helps to slow database growth as every L2 block includes a L1 Attributes
 deposit transaction.
 
+In the first L2 block after the Ecotone activation block, the Ecotone L1 attributes are first used.
+
+The pre-Ecotone values are migrated over 1:1. The new Ecotone attributes are set as following:
+
+- The `baseFeeScalar` is set to the pre-Ecotone `scalar` value.
+- The `blobBaseFeeScalar` is set to 0.
+- The pre-Ecotone `overhead` attribute is dropped.
+- The `blobBaseFee` represents the L1 blob basefee of the L1 origin block.
+  Or `0` if the L1 block does not support blobs.
```

Comment:
oh hmm I think the existing implementation will actually panic on nil pointer exception if the L1 block doesn't support blobs (since BlobBaseFee returns nil for such blocks).  Do you want to fix that here or should I submit a PR?  You'd just have to add the nil check to L270 of op-node/rollup/derive/l1_block_info.go.

Reply:
Yes, I am planning on handling this case in the implementation.